monkey/common/agent_plugins/agent_plugin_type.py

```diff
@@ -2,7 +2,7 @@
 
 
 class AgentPluginType(Enum):
-    CREDENTIAL_COLLECTOR = "CredentialCollector"
+    CREDENTIAL_COLLECTOR = "Credential_Collector"
```

Comment:
Why is this change necessary?

Reply:
Because of this:
<img width="738" alt="image" src="https://user-images.githubusercontent.com/15820737/234197359-86cbae3b-73de-4e5b-b647-9c177f727c10.png">

The name and the value of the `Credential Collector` are not the same but they should be cause it won't find the correct plugin type

Reply:
https://github.com/guardicore/monkey/pull/3247#discussion_r1176169986